index.d.ts

```diff
@@ -1,3 +1,16 @@
+interface Options {
```

Comment:
This is not correctly exported.

Reply:
This was part of the original PR but was only pointed out now?  This doesn't need to be exported, like a function does. These are options and are a part of the exported function.